site/static/error.html

```diff
@@ -131,6 +168,17 @@ <h1>
         .Error.Title }}
       </h1>
       <p>{{ .Error.Description }}</p>
+      {{- if .Error.Warnings }}
+      <div class="warning">
+        <div class="warning-title">
+          <svg  height="1em" width="auto" focusable="false" aria-hidden="true" viewBox="0 0 24 24"><path fill="#e66828" d="M12 5.99L19.53 19H4.47L12 5.99M12 2L1 21h22L12 2zm1 14h-2v2h2v-2zm0-6h-2v4h2v-4z"></path></svg>
+          <h3>Warnings</h3>
+        </div>
+        {{ range $i, $v := .Error.Warnings }}
+          <p>- {{ $v }}</p>
```

Comment:
I think this would look better with `<ul>{{ range }}<li>...</li>{{ end }}</ul>` instead.

Reply:
I originally did that, but then I lose the default text styling.

I can put back to that and copy the text styling.